pkg/apis/configuration/validation/policy.go

```diff
@@ -7,18 +7,18 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/nginxinc/kubernetes-ingress/pkg/apis/configuration/v1alpha1"
+	v1 "github.com/nginxinc/kubernetes-ingress/pkg/apis/configuration/v1"
```

Comment:
Is `v1` at the start of this line unnecessary, since the package is named `v1` anyway?

Reply:
This comes directly from running `goimports`

If I remove it and run `goimports pkg/apis/configuration/validation/policy.go -w` again it will re-add it.

My guess is that `goimports` does this with small pkg names as it is more likely that you'll come across import collisions. And this would make it more clear and convenient to change an import alias as you see fit?